go/Makefile

```diff
@@ -23,7 +23,7 @@ fmt: $(MODULES)
 lint: $(MODULES)
 	@for f in $(^D); do \
 	(cd $$f; echo "Checking golangci-lint $$f"; \
-	(which $(GOPATH)/bin/golangci-lint || go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.44.2); \
+	(which $(GOPATH)/bin/golangci-lint || go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.50.1); \
```

Comment:
Nit: I really like using `go run` where it works - it seems plenty fast enough now.  It also works really nicely with go generate.

(No need to fix, just one for the future!)

Reply:
Thank you! Yeah, `go run` is more proper in this case in a follow-up PR.